lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -183,6 +183,10 @@ def pull_tests_from_queue(can_initialize_queue: false)
             executed_test_files: @node_test_file_paths
           )
           @node_test_file_paths += test_file_paths
+
+          time_tracker = KnapsackPro::Formatters::TimeTrackerFetcher.call
+          time_tracker.scheduled_test_file_paths = @node_test_file_paths
```

Comment:
Each time we set what was there before plus the new batch?

Reply:
Yes.

Each time we override the time_tracker.scheduled_test_file_paths with @node_test_file_paths.

The @node_test_file_paths is updated on each batch pulled from the Queue API.